bciers/apps/administration/tests/components/operations/OperationDataGrid.test.tsx

```diff
@@ -101,7 +97,6 @@ describe("OperationsDataGrid component", () => {
     expect(
       screen.getByRole("columnheader", { name: "BC GHG ID" }),
     ).toBeVisible();
-    expect(screen.getByRole("columnheader", { name: "Status" })).toBeVisible();
```

Comment:
We still see status for external users, so any status assertions for external users can stay

Reply:
Good catch, thanks!